### PR TITLE
Fix 359

### PR DESCRIPTION
**Pull request by groolot**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -124,4 +125,4 @@ run-tests: luakit luakit.so tests/util.so
 	@$(LUA_BIN_NAME) tests/run_test.lua
 
 newline: options;@echo
-.PHONY: all clean options install newline apidoc doc default
+.PHONY: all clean options install newline apidoc doc
```
Comment (aidanholm) [inline Makefile:128]: `default` should still be a phony target

**Review by aidanholm — CHANGES_REQUESTED** [review]
Please use a more descriptive commit message, e.g. `Deprecate default target`.
If you wish to add the issue number, please add it to the revert commit message, e.g. `Revert "makefile: add default target" (#359)`.
Thanks!

File: Makefile (hunk 1 above)
Comment (groolot) [reply Makefile:128]: Yes it's true. default need to be .PHONY

**Comment by aidanholm** [comment]
@groolot In case you're unaware; you can amend your commit and (force) push to `groolot:fix-359`, and this PR will be updated.

**Pull request closed without merging**